src/client/sandbox/node/window.js

```diff
@@ -326,6 +326,15 @@ export default class WindowSandbox extends SandboxBase {
             };
         }
 
+        if (window.DOMParser) {
+            window.DOMParser.prototype.parseFromString = function (...args) {
+                if (args.length > 1 && typeof args[0] === 'string')
```

Comment:
I'm disagree with
> args.length is an uncessary check. Drop this.

The `parseFromString` function throw an error `Failed to execute 'parseFromString' on 'DOMParser': 2 arguments required, but only 1 present.` and we don't need to process the string.